google-cloud-examples/src/test/java/com/google/cloud/examples/storage/snippets/ITBucketSnippets.java

```diff
@@ -133,4 +139,50 @@ public void testBucket() throws InterruptedException {
     thrown.expect(StorageException.class);
     assertTrue(bucketSnippets.delete());
   }
+
+  @Test
+  public void testListBucketIamMembers() {
+    // Test an added Bucket-level IAM member is listed 
+    Policy policy = storage.getIamPolicy(BUCKET);
+    storage.setIamPolicy(BUCKET, policy.toBuilder().removeRole(StorageRoles.admin()).build());
+    policy = storage.getIamPolicy(BUCKET);
+    assertNull(policy.getBindings().get(StorageRoles.admin()));
+    storage.setIamPolicy(BUCKET, policy.toBuilder().addIdentity(StorageRoles.admin(),
+            Identity.user(USER_EMAIL)).build());
+    policy = storage.getIamPolicy(BUCKET);
```

Comment:
You can get the resultant policy from the setIamPolicy, but you can't expect it to always come back from storage.getIamPolicy() immediately following.